src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -530,11 +536,32 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     updateReplies = (id: string, replies: Array<Comment>) => {
-        const { api, file } = this.props;
+        const { activeFeedEntryId, api, file, history } = this.props;
+        const { feedItems } = this.state;
 
-        const feedAPI = api.getFeedAPI(false);
+        if (!feedItems) {
+            return;
+        }
 
+        const feedAPI = api.getFeedAPI(false);
         feedAPI.file = file;
+
+        // Detect if replies are being hidden and activeFeedEntryId belongs to a reply
+        // that is in currently being updated parent, in order to disable active item
+        if (
+            activeFeedEntryId &&
+            replies.length === 1 &&
```

Comment:
Would this cause any issues with threads that are exactly one reply long?

Reply:
It should not as threads can be "hidden" only if there are 2 or more replies. Hiding replies basically means setting replies array to just one entry (the last reply).